### ISSUE TITLE
Evaluating dataset outside training

### ISSUE THREAD

**Issue by hetryn** [ThilinaRajapakse/simpletransformers#702]
Is there a dedicated function to evaluate test set outside of training?

i.e. after saving the trained model, call the function to evaluate the model on any test set without training again.

**Comment by AbinayaM02** [comment]
@hetryn : You can just load the model and use model.predict([test data]) to get the predictions. This way you can evaluate the model without training. For getting the metric (e.x. confusion matrix or so), you can use scikit-learn function based on what metric you want to evaluate on.
